### ISSUE TITLE
julia 0.5.0: build error: No rule to make target 'include/mxnet/symbolic.h'

### ISSUE THREAD

**Issue by jbrea** [dmlc/MXNet.jl#201]
I get the following error after ```Pkg.add("MXNet")```
(which is commit e599d51 of MXNet.jl)

```
make: *** No rule to make target 'include/mxnet/symbolic.h', needed by 'build/src/c_api/c_predict_api.o'.  Stop.
=======================================[ ERROR: MXNet ]========================================

LoadError: failed process: Process(`make -j1 USE_BLAS=openblas 'MSHADOW_LDFLAGS=-lm /usr/bin/../lib/julia/libopenblas.so'`, ProcessExited(2)) [2]
while loading /mnt/ubuntu/home/j/.julia/v0.5/MXNet/deps/build.jl, in expression starting on line 27

===============================================================================================
```

**Julia Version 0.5.0**
```
Commit 3c9d753 (2016-09-19 18:14 UTC)
Platform Info:
  System: Linux (x86_64-pc-linux-gnu)
  CPU: Intel(R) Core(TM) i7-5600U CPU @ 2.60GHz
  WORD_SIZE: 64
  BLAS: libopenblas (DYNAMIC_ARCH NO_AFFINITY Haswell)
  LAPACK: libopenblas
  LIBM: libm
  LLVM: libLLVM-3.7.1 (ORCJIT, broadwell)
```

**Comment by MikeInnes** [comment]
I've been seeing errors similar to this, and what seems to work is checking out an older version (say v0.1.0), building that, then checking out the newer version and rebuilding. Not ideal but appears to get the job done.

**Comment by MikeInnes** [comment]
Ok, perhaps not; this got me past the first header error, but now I get `make: *** No rule to make target `include/mxnet/symbolic.h', needed by `build/src/c_api/c_api.o'.  Stop.`.

**Comment by jbrea** [comment]
Manual compilation of MXNet works for me (see http://dmlc.ml/MXNet.jl/latest/user-guide/install/#Manual-Compilation-1)

**Comment by vchuravy** [comment]
I think this might have to do with the submodules and changed dependencies. What is your MXNet commit?

**Comment by vchuravy** [comment]
Should be fixed by c4a2658, the issue was that a new submodule was added, but we did not initialise it.
